.github/workflows/benchmark-runtime-weights.yml

```diff
@@ -3,14 +3,21 @@ name: Benchmark runtime weights
 on:
   workflow_dispatch:
     inputs:
-      chain:
-        type: choice
-        description: The chain whose runtime is benchmarked, all = rococo+litmus+litentry
-        options:
-          - all
-          - litmus
-          - litentry
-          - rococo
+      litentry:
+        type: boolean
+        description: The litentry will be benchmarked
```

Comment:
Can we simply write "litentry/litmus/rococo" to be less verbose?

Reply:
Sorry, I can't find any documentation on github that can define the same type of input in one line

Reply:
I simply mean we don't need to write "will be benchmarked" as it's in a benchmarking GHA by itself 😂 
but it's OK